doc/source/qip-processor.rst

```diff
@@ -159,19 +159,19 @@ To let it find the optimal pulses, we need to give the parameters for :func:`~qu
 
 
       # Same parameter for all the gates
-      qc = QubitCircuit(N=1)
+      qc = QubitCircuit(1)
```

Comment:
```suggestion
      qc = QubitCircuit(1)
```
just a question, why did you delete "N=" here and elsewhere?

Reply:
I'm trying to discourage the use of capital `N` as the number of qubits, and eventually replace it with `num_qubits`, to be consistent.

But maybe this should not be included here in this PR, I agree. I can revert it if desired.